flags/src/main/java/com/yahoo/vespa/flags/Flags.java

```diff
@@ -241,7 +241,13 @@
     public static final UnboundBooleanFlag NGINX_UPSTREAM_PROXY_PROTOCOL = defineFeatureFlag(
             "nginx-upstream-proxy-protocol", false,
             "Whether the nginx should enable proxy-protocol for all upstreams",
-            "Takes effect immediately");
+            "Takes effect immediately",
+            ZONE_ID, APPLICATION_ID);
+
+    public static final UnboundBooleanFlag NLB_PROXY_PROTOCOL = defineFeatureFlag(
+            "nlb-proxy-protocol", false,
+            "Configure NLB to use proxy protocol",
+            "Takes effect on next application redeploy");
```

Comment:
No dimensions?

Reply:
Nor sure how that was removed... should definitely have application_id dimension